File: Modul1/Python/PRAK110-2310817210023-RifkyPutraMahardika.py
import math
Alas=5; Tinggi=12
Miring=round(math.sqrt(pow(Alas, 2)+pow(Tinggi, 2)))
Keliling=round(Tinggi+Alas+Miring)
Luas=round((Alas*Tinggi)/2)
print("Diketahui :")
print(f"Alas = {Alas} cm")
print(f"Tinggi = {Tinggi} cm")
print("Jawab :")
print(f"Sisi A = {Tinggi} cm")
print(f"Sisi B = {Miring} cm")
print(f"Sisi C = {Alas} cm")
print(f"Keliling = {Keliling}")
print(f"Luas = {Luas} cm")

File: Modul1/C/PRAK110-2310817210023-RifkyPutraMahardika.c
#include<stdio.h>
#include<math.h>
int main(){
    int Alas = 5, Tinggi = 12;
    float Miring = sqrt(pow(Alas, 2) + pow(Tinggi, 2));
    float Keliling = Tinggi+Alas+Miring;
    float Luas = (Alas*Tinggi)/2;

    printf("Diketahui : \n");
    printf("Alas = %d cm\n", Alas);
    printf("Tinggi = %d cm\n", Tinggi);
    printf("Jawab : \n");
    printf("Sisi A = %d cm\n", Tinggi);
    printf("Sisi B = %.f cm\n", Miring);
    printf("Sisi C = %d cm\n", Alas);
    printf("Keliling = %.f cm\n", Keliling);
    printf("Luas = %.f cm\n", Luas);

return 0;
}